src/crypto/hasher.rs

```diff
@@ -1,114 +1,86 @@
-use super::blake3_sequence::Blake3SeqNo;
+use super::address::CryptoAddress;
 use crate::prelude::*;
-use crate::stable_hash::UnorderedAggregator;
 use blake3::Hasher;
 use ibig::UBig;
 use lazy_static::lazy_static;
 use num_traits::identities::One;
 use std::default::Default;
 
+// TODO: Consider using a Solinas prime
+/*
+From Jackson:
+    In particular we could change the prime to be a Solinas prime.
+    If we implement the algorithm for fast modular multiplication around a solinas prime then we get a big speed up.
+    So the changes would be just change the public parameter prime in the codebase, and don’t just naively multiply and reduce, but write the algorithm to take advantage of the structure inherent in Solinas primes
+    (they are prime numbers that have really low hamming weights, a sort of generalization of Mersenne primes — and so computers love these numbers)
```

Comment:
Possibly, though it will no longer be relevant to graph-node. So, it's not a priority for me but it someone ever decides to use the crypto version for some reason in the future the option will be open to them.